examples/react-widget-app/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
replace this and use https://github.com/ChainSafe/eslint-config?tab=readme-ov-file#frontend-react